skylarktest/skylarktest.go

```diff
@@ -16,11 +16,13 @@ import (
 	"bytes"
 	"fmt"
 	"go/build"
+	"os"
 	"path/filepath"
 	"regexp"
 	"sync"
 
 	"github.com/google/skylark"
+	"github.com/google/skylark/bazeltest"
```

Comment:
The purpose of making DataFile a variable containing a function is that you can override it during initialization of your application. There's no need for an "upwards" dependency from this package to bazeltest.  Instead, get your bazel build file to compile this additional file into `skylarktest`:

```
package skylarktest

func init() {
	DataFile = func(pkgdir, filename string) string {
		return filepath.Join("your/bazel/dir", pkgdir, filename)
	}
}
```

Reply:
•https://github.com/google/skylark/pull/87.patch